src/v2/view-builder/utils/ChallengeViewUtil.js

```diff
@@ -14,8 +14,15 @@ import hbs from 'handlebars-inline-precompile';
 import Enums from '../../../util/Enums';
 import Util from '../../../util/Util';
 
+export function appendLoginHint(deviceChallengeUrl, loginHint) {
```

Comment:
+1 for this

Reply:
@pradeepdewda-okta  Move `appendLoginHint` to page object as well? The function is a util.

Reply:
No. it's already in a util file `ChallengeViewUtil`

Reply:
Page object is only for testcafe files